pkg/web/handlers.go

```diff
@@ -1,8 +1,13 @@
 package web
 
 import (
+	"fmt"
 	"net/http"
 
+	"net/url"
+
```

Comment:
No need for empty lines between imported  libs, only one empty line between stdlib and 3rd party imports - for some reasons `goimports` do this since some time, previously it was working fine.